### PR TITLE
[Relay/TOPI][Op] Add variance and layer norm op

### PR DESCRIPTION
**Pull request by icemelon9**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

@junrushao1994 @kevinthesun Could you have help review?

### CODE REVIEW

**Comment by icemelon9** [comment]
@kevinthesun @masahi please help to take a look.

File: python/tvm/relay/op/nn/nn.py [hunk 1]
```diff
@@ -897,6 +897,64 @@ def batch_norm(data,
...
+        The gamma scale factor.
+
+    beta : tvm.relay.Expr
+        The beta offset factor.
+
+    axis : int, optional, default=-1
+        The axis that should be normalized, typically the axis of the channels.
+
+    epsilon : double, optional, default=1e-5
+        Small float added to variance to avoid diving by zero.
```
Comment (masahi) [inline python/tvm/relay/op/nn/nn.py]: dividing

File: python/tvm/relay/op/reduce.py [hunk 2]
```diff
@@ -257,6 +260,140 @@ def mean(data, axis=None, keepdims=False, exclude=False):
...
+def variance(data, axis=None, keepdims=False, exclude=False):
+    """Computes the variance of data over given axes.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data
+
+    axis : None or int or tuple of int
+        Axis or axes along which a mean operation is performed.
```
Comment (masahi) [inline python/tvm/relay/op/reduce.py]: mean -> variance

File: python/tvm/relay/op/reduce.py [hunk 3]
```diff
@@ -257,6 +260,140 @@ def mean(data, axis=None, keepdims=False, exclude=False):
...
+        The default, axis=None, will find the indices of minimum element all of the elements of
```
Comment (masahi) [inline python/tvm/relay/op/reduce.py]: all of -> of all

File: python/tvm/relay/op/reduce.py [hunk 4]
```diff
@@ -257,6 +260,140 @@ def mean(data, axis=None, keepdims=False, exclude=False):
...
+def std(data, axis=None, keepdims=False, exclude=False):
+    """Computes the standard deviation of data over given axes.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data
+
+    axis : None or int or tuple of int
+        Axis or axes along which a mean operation is performed.
```
Comment (masahi) [inline python/tvm/relay/op/reduce.py]: mean -> standard deviation

File: python/tvm/relay/op/reduce.py [hunk 5]
```diff
@@ -257,6 +260,140 @@ def mean(data, axis=None, keepdims=False, exclude=False):
...
+        The default, axis=None, will find the indices of minimum element all of the elements of
```
Comment (masahi) [inline python/tvm/relay/op/reduce.py]: of all

File: python/tvm/relay/op/reduce.py [hunk 6]
```diff
@@ -257,6 +260,140 @@ def mean(data, axis=None, keepdims=False, exclude=False):
...
+    """Computes the mean and variance of data over given axes.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data
+
+    axis : None or int or tuple of int
+        Axis or axes along which a mean operation is performed.
+        The default, axis=None, will find the indices of minimum element all of the elements of
```
Comment (masahi) [inline python/tvm/relay/op/reduce.py]: of all

**Comment by icemelon9** [comment]
Thanks @masahi . The docs are updated.

**Comment by masahi** [comment]
thanks @icemelon9 @junrushao1994 this is merged.

**Pull request merged**